<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Todo</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>350</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Tools</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Text :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>350</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time :</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>145</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QFontComboBox" name="fontComboBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Font :</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View GitHub</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>361</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Languages</string>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>English</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Russian</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Georgian</string>
       </property>
      </widget>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>MIT License</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
